dags/utils/status.py

```diff
@@ -1,4 +1,8 @@
+import logging
+
+from airflow.operators.python_operator import PythonOperator
 from airflow.operators.dataset_status import DatasetStatusOperator
+from airflow.hooks.dataset_status import DatasetStatusHook
```

Comment:
tests are failing with ImportError: No module named dataset_status

Reply:
It looks like my unittests are a bit brittle in this section, I'll give it a look over and re-tag you once I figure out how to untangle this.

Reply:
re: this is part of the unittest that I fixed where imports are relative to the root path. Airflow edits the sys import paths for the plugins so they look consistent.